core/services/job/models.go

```diff
@@ -121,7 +121,7 @@ type Job struct {
 	BlockhashStoreSpec   *BlockhashStoreSpec
 	BootstrapSpec        *BootstrapSpec
 	BootstrapSpecID      *int32
-	PipelineSpecID       int32
+	PipelineSpecID       *int32
```

Comment:
why did we make this change? we didn't make the value nullable in the DB. I thought we landed on "keep the value non-nullable, and just make keepers observation source blank".